package.json

```diff
@@ -122,6 +123,7 @@
     "react-test-renderer": "^15.5.4",
     "rimraf": "^2.6.0",
     "sass-loader": "^6.0.2",
+    "selenium-server-standalone-jar": "^3.4.0",
```

Comment:
use a fixed version, if a newer version gets installed, then `server_path` will be incorrect. 
`"selenium-server-standalone-jar": "3.4.0",`